engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/EpbContext.java

```diff
@@ -101,4 +106,43 @@ final void initializePolyfill(Node node, TruffleContext ctx) {
       WebEnvironment.initialize(eval, exec);
     }
   }
+
+  final void handleMultiAccess(String msg) {
+    try {
+      var ms = delayer.nextInt(10, 1000);
```

Comment:
shouldn't we log this event **always**?

Reply:
- I was thinking about some heuristics to decide when to log, but 
- at the end I decided to leave it out of this PR
- I am not sure logging (and dumping all the thread stacks) always is what we want 
- there might be plenty of events like this
- a one line note in the log when verbosity is high might be OK
- so far I haven't seen many clashes even I am running with `-ea` on